test/lit/passes/merge-blocks.wast

```diff
@@ -71,11 +73,13 @@
  ;; CHECK:      (func $struct.get
  ;; CHECK-NEXT:  (nop)
  ;; CHECK-NEXT:  (drop
- ;; CHECK-NEXT:   (i32.const 1234)
- ;; CHECK-NEXT:  )
- ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (struct.get $struct 0
- ;; CHECK-NEXT:    (ref.null $struct)
+ ;; CHECK-NEXT:    (block (result (ref null $struct))
+ ;; CHECK-NEXT:     (drop
+ ;; CHECK-NEXT:      (i32.const 1234)
+ ;; CHECK-NEXT:     )
+ ;; CHECK-NEXT:     (ref.null none)
+ ;; CHECK-NEXT:    )
```

Comment:
This is no longer being optimized properly. MergeBlocks should have moved the nested drop to the outer block. Perhaps the pass is confused by the type of the null?